azure-devops/global/abstracts/script-e2e-tests.yml

```diff
@@ -0,0 +1,105 @@
+parameters:
+  - name: TAG_NAME
+    type: string
+  - name: PLUGIN_NAME
+    type: string
+
+steps:
+  - task: NodeTool@0
```

Comment:
I know you were testing, but now I think you can leverage the code already existing. Right?!